pkg/command/executor.go

```diff
@@ -61,8 +58,17 @@ func (c *command) Exec() (out []byte, err error) {
 		}
 	}()
 
-	clog.Debug("execute", "command", cmd.Args, "dir", cmd.Dir)
-	if c.ioConnected {
+	out, err = e.exec(ctx, c, cmd)
+
+	signal.Reset()
+	close(sigCh)
+
+	wg.Wait()
+	return
```

Comment:
**[wraperr]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
	github.com/izumin5210/grapi/pkg/command.Exec

Reply:
fixed in 1e9fd43